CondCore/EcalPlugins/src/plugin.cc

```diff
@@ -172,6 +175,7 @@ REGISTER_PLUGIN(EcalIntercalibErrorsRcd, EcalCondObjectContainer<float>);
 REGISTER_PLUGIN(EcalADCToGeVConstantRcd, EcalADCToGeVConstant);
 REGISTER_PLUGIN(EcalLaserAlphasRcd, EcalCondObjectContainer<float>);
 REGISTER_PLUGIN(EcalLaserAPDPNRatiosRcd, EcalLaserAPDPNRatios);
+REGISTER_PLUGIN(EcalLaserAPDPNRatiosMCRcd, EcalLaserAPDPNRatiosMC);
```

Comment:
@amassiro   this line is instructing the Framework to read the payloads of the tag "EcalLaserAPDPNRatios_UL_2017_mc" as "EcalLaserAPDPNRatiosMC" objects. No way to make this working.